src/SmartComponents/CompletedTaskDetailsKebab/CompletedTaskDetailsKebab.js

```diff
@@ -1,53 +1,65 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import propTypes from 'prop-types';
 import {
   Dropdown,
   DropdownItem,
-  DropdownPosition,
-  KebabToggle,
-} from '@patternfly/react-core';
+  DropdownList,
+  MenuToggle,
+} from '@patternfly/react-core/';
+import { EllipsisVIcon } from '@patternfly/react-icons';
 
-const CompletedTaskDetailsKebab = ({ status, setModalOpened }) => {
-  const [isOpen, setIsOpen] = useState(false);
-  const createDropdownItems = () => {
-    //let type = status === 'Running' ? 'cancel' : 'delete';
-    let type = 'delete';
-    return [
-      <DropdownItem
-        aria-label={`${type}-task-kebab-button`}
-        key={`${type}-task`}
-        component="button"
-        data-ouia-component-id={`${type}-task-dropdown-item`}
-        onClick={() => setModalOpened(true)}
-        isDisabled={status !== 'Completed'}
-      >
-        {type[0].toUpperCase() + type.slice(1)}
-      </DropdownItem>,
-    ];
-  };
+const CompletedTaskDetailsKebabItems = ({ status, setModalOpened }) => {
+  //let type = status === 'Running' ? 'cancel' : 'delete';
+  let type = 'delete';
 
-  const [dropdownItems, setDropdownItems] = useState(createDropdownItems());
+  return (
+    <DropdownItem
+      aria-label={`${type}-task-kebab-button`}
+      key={`${type}-task`}
+      component="button"
+      data-ouia-component-id={`${type}-task-dropdown-item`}
+      onClick={() => setModalOpened(true)}
+      isDisabled={status !== 'Completed'}
+    >
+      {type[0].toUpperCase() + type.slice(1)}
+    </DropdownItem>
+  );
+};
 
-  useEffect(() => {
-    if (status !== undefined) {
-      setDropdownItems(createDropdownItems());
-    }
-  }, [status]);
+CompletedTaskDetailsKebabItems.propTypes = {
+  setModalOpened: propTypes.func,
+  status: propTypes.string,
+};
+
+const CompletedTaskDetailsKebab = ({ status, setModalOpened }) => {
+  const [isOpen, setIsOpen] = useState(false);
 
   return (
     <Dropdown
       onSelect={() => setIsOpen(false)}
-      position={DropdownPosition.right}
-      toggle={
-        <KebabToggle
-          onToggle={() => setIsOpen(!isOpen)}
+      toggle={(toggleRef) => (
+        <MenuToggle
+          ref={toggleRef}
           id="executed-task-kebab"
-        />
-      }
+          aria-label="kebab dropdown toggle"
```

Comment:
The `aria-label` should be something more descriptive and specific, since this is what screen readers may read. In this case something like "Task Details Menu Toggle".